### PR TITLE
Removed hard coding of "rrobin" and replaced with constant variables

### LINKED ISSUE #113: Systematically Use Constant Strings in Different Modules
The source code for different modules contains hard-coded strings! For instance, the main `gatorgrouper.py` file contains the following source code:

```
# generate the groups and display them
    if GG_ARGUMENTS.grouping_method == "rrobin" and GG_ARGUMENTS.num_group is DEFAULT_NUMGRP:
        GROUPED_STUDENT_IDENTIFIERS = group_rrobin_group_size(
            SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.group_size)
    elif GG_ARGUMENTS.grouping_method == "rrobin" and GG_ARGUMENTS.num_group is not DEFAULT_NUMGRP:
        GROUPED_STUDENT_IDENTIFIERS = group_rrobin_num_group(
            SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.num_group)
    elif GG_ARGUMENTS.num_group is DEFAULT_NUMGRP:  # default to random method
        GROUPED_STUDENT_IDENTIFIERS = group_random_group_size(
            SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.group_size)
    else:
        GROUPED_STUDENT_IDENTIFIERS = group_random_num_group(
SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.num_group)
```

I suggest that we start to avoid using hard-coded strings like `rrobin` and instead use constant variables. Also, I suggest that the names of the constant variables be much more descriptive.

### PR DESCRIPTION
**Pull request by baldeosinghm**
## Removal of hard coding of "rrobin" for constant variable

This pull request will remove any hard coding of the string "rrobin" in all of gatorgrouper.  Gatorgrouper now uses global.py, a file containing constant variable that hold the string "rrobin".

Fixes #113 

## Type of Change: Hard Coding Removal

Please describe the pull request as one of the following: 

- [ ] Bug fix
- [ ] Breaking change
- [ ] New feature
- [ ] Documentation update
- [x] Other (Please specify any helpful information below)

Changed hard coding of string "rrobin" and instead used constant variable ALGORITHM_ROUND_ROBIN from globals.py.

## Tags

Matthew Baldeosingh

### CODE REVIEW

**Comment by Michionlion** [comment]
> There's now a merge conflict, but it looks good otherwise.

@baldeosinghm If you merge `master` into your branch (with the `Resolve conflicts` button on Github), and everything looks the same, then I'll merge this since it has the required reviews.

**Comment by Michionlion** [comment]
@baldeosinghm Just some minor linting issues to fix:

```
gatorgrouper/globals.py:1:0: C0111: Missing module docstring (missing-docstring)
gatorgrouper/utils/gatorgrouper.py:43:38: E0602: Undefined variable 'DEFAULT_NUMGRP' (undefined-variable)
gatorgrouper/utils/gatorgrouper.py:50:42: E0602: Undefined variable 'DEFAULT_NUMGRP' (undefined-variable)
```

File: gatorgrouper/utils/parse_arguments.py [hunk 1]
```diff
@@ -70,7 +69,7 @@ def parse_arguments(args):
         help="Use round-robin grouping method",
         action="store_const",
         dest="grouping_method",
-        const=ALGORITHM_ROUND_ROBIN,
+        const="rrobin",
```
Comment (Michionlion) [inline gatorgrouper/utils/parse_arguments.py]: This must use the `constants` module, that is the whole reason to have it.

**Comment by baldeosinghm** [comment]
Fixed all necessary changes and merge conflict

**Pull request merged**